.github/workflows/build-plugin-zip.yml

```
@@ -93,10 +93,10 @@ jobs:
                       fi
                     fi
                   fi
-                  echo "::set-output name=new_version::$(echo $NEW_VERSION)"
+                  echo "new_versio=$(echo $NEW_VERSION)" >> $GITHUB_OUTPUT
                   IFS='.' read -r -a NEW_VERSION_ARRAY <<< "$NEW_VERSION"
                   RELEASE_BRANCH="release/${NEW_VERSION_ARRAY[0]}.${NEW_VERSION_ARRAY[1]}"
-                  echo "::set-output name=release_branch::$(echo $RELEASE_BRANCH)"
+                  echo "release_branch=$(echo $RELEASE_BRANCH)" >> $GITHUB_OUTPUT
```

Another double echo.

```suggestion
                  echo "release_branch=${RELEASE_BRANCH}" >> $GITHUB_OUTPUT
```